src/link/resolvePeers.ts

```diff
@@ -53,6 +53,8 @@ export type DependencyTreeNode = {
     cpu?: string[],
     os?: string[],
   },
+  sideEffectsCache: Map<string, string>,
+  importedFromCache?: boolean,
```

Comment:
Might be better to be more specific here, so `importedFromSideEffectsCache` or `isBuilt` or `skipScripts`